RELEASE_DOCS.md

```diff
@@ -0,0 +1,16 @@
+# How to release collins
+
+- [ ] Go to https://github.com/tumblr/collins/releases and click "Draft a new release"
+- [ ] The tag version should be `v<next version number>` according to [semantic versioning](http://semver.org/).  The release title should be the same version number but without the leading `v`.  For example, the tag for version 2.1.0 is [`v2.1.0`](https://github.com/tumblr/collins/tree/v2.1.0) and the release is titled [`2.1.0`](https://github.com/tumblr/collins/releases/tag/v2.1.0).
+- [ ] Make sure the release is set to target master.
+- [ ] Type a few paragraphs highlighting major features or changes in this release.
+- [ ] Run `scripts/gen_changelog.rb` to generate a changelog of PRs from the last release.  Copy what it prints and paste it at the bottom of the release draft.
+- [ ] Check out the collins code in an environment running jdk 7 with the unlimited strength JCE configured correctly, and play activator checked out in the correct location (`$HOME/src/activator-1.3.6-minimal/activator`).  Run `scripts/package.sh` and attach the resulting zip to the release, and name it `collins-v<version number>.zip`.  
+- [ ] Do the previous step but in an environment with jdk 8.  Attach the zip to the release, and title it `collins-v<version number>-jdk8.zip`.
+- [ ] Save, but do not publish, the draft.  If you want to get proofreading for your release text, this is the right time to do it.
+- [ ] Cut a new branch off of master.  Copy the text of the release into the [CHANGELOG.md](https://github.com/tumblr/collins/blob/master/CHANGELOG.md) file.  Open a PR and get thumbs.  See https://github.com/tumblr/collins/pull/490 for an example PR.
```

Comment:
Yeah I left it in there since I likely will not be running the next release - if that's the way we want to go, then whomever is doing it next can make those changes.